### PR TITLE
Adding missing ECDsaCng members

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/corefx/issues/17163

@danmosemsft See open question in linked issue

### CODE REVIEW

**Comment by ViktorHofer** [comment]
Added the two members which throw PNSE.

File: src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.cs [hunk 1]
```diff
@@ -2,13 +2,30 @@
...
+        /// </summary>
+        public CngAlgorithm HashAlgorithm
+        {
+            get
+            {
+                return _hashAlgorithm;
+            }
+            set
+            {
+                _hashAlgorithm = value ?? throw new ArgumentNullException("value");
```
Comment (stephentoub) [inline src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.cs]: Nit: nameof(value)

**Comment by bartonjs** [comment]
@dotnet-bot Test Innerloop Ubuntu14.04 Debug x64 Build and Test please

**Comment by bartonjs** [comment]
LGTM with squash

**Pull request merged**